code/style.css

```diff
@@ -1,4 +1,207 @@
-html {
-  background: #0025ff;
-  color: #fff;
+/*These style the main elements of entire website*/
+
+body {
+  margin: 0;
+  font-family: 'Cormorant', serif;
+  font-family: 'Cormorant Garamond', serif;
+  font-family: 'Spectral', serif;
+  background-image: url("background.png");
 }
+
+main {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+
+h1 {
+  font-size: 60px;
+  margin-top: 25px;
+  margin-bottom: 0;
+}
+
+h3 {
+  font-style: italic;
+}
+
+h4 {
+  font-size: 15px;
+  margin: 0;
+
+}
+
+h5 {
+  font-size: 12px;
+  margin: 0;
+}
+
+/*These style the header and its content*/
+
+header {
+  display: flex;
+  align-items: center;
+  justify-content: space-evenly;
+}
+
+ul {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  display: flex;
+  justify-content: space-evenly;
+}
+
+nav {
+  width: 40%;
+}
+
+li {
+  font-size: 12px;
+}
+
+li:hover{
+    -webkit-transform: scale(1.2);
+    -ms-transform: scale(1.2);
+    transform: scale(1.2);
+}
+
+.logo {
+  padding: 3%;
+  width: 8%;
+}
+
+
+.logo img {
+  border-radius: 90%;
+  width: 100%;
+  animation: pulse 5s infinite;
+}
+
+@keyframes pulse {
+  0% {
+    background-color: #fcfcfc;
+  }
+  100% {
+    background-color: #5574db;
+  }
+}
+
+/*These style the home image and text*/
+
+.home {
+  display: flex;
+  flex-direction: column;
+  width: 100vw;
+  align-items: center;
+  flex-wrap: wrap;
+}
+
+.big-image img {
+  width: 90vw;
+  border-radius: 75%;
+  margin: 5%;
+}
+
+.big-text {
+  display:flex;
+  flex-direction: column;
+  width: 100%;
+  text-align: center;
+}
+
+/*These style the articles and their content*/
+
+.news-box {
+  display: flex;
+  text-align: center;
+  flex-direction: column;
+  margin: 5% 0%;
+  border: 1px solid black;
+  width: 50%;
+  box-shadow: rgba(50, 50, 93, 0.25) 0px 50px 100px -20px, rgba(0, 0, 0, 0.3) 0px 30px 60px -30px, rgba(10, 37, 64, 0.35) 0px -2px 6px 0px inset;
+}
+
+.news {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-evenly;
+  margin-top: 5%;
+}
+
+.news-box img {
+  width: 100%;
+}
+
+.news-box img:hover{
+  opacity:0.5;
+}
+
+.big-text {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  align-items: center;
+}
+
+/*These style the footer OBS! navbar styling in header*/
+
+footer {
+  display: flex;
+  align-items: center;
+  justify-content: space-evenly;
+}
+
+
+@media (max-width: 667px) {
```

Comment:
I found this article https://www.freecodecamp.org/news/taking-the-right-approach-to-responsive-web-design/ regarding mobile-first design, where they encourage usage of min-width - and I see that both are in use in the code, not 100% sure but think it's encouraged to use min-width from mobile breakpoints and counting upwards to desktop breakpoints.